azavea/raster-vision
# Migrate from Travis to GitHub Actions

## Overview

Ports Travis CI config to a GitHub Actions workflow.

### Checklist


## Testing Instructions

 * Verify that the [GitHub Actions CI workflow](https://github.com/azavea/raster-vision/runs/3146625884) passes.
* Verify that [this Release workflow](https://github.com/azavea/raster-vision/runs/3146630550) ran.
* Verify that the Release workflow has uploaded a working Docker image to Quay.io:
  ```#bash
  docker pull quay.io/azavea/raster-vision:pytorch-test-gha
  ```
* Run `./scripts/test` locally and confirm that enabled tests run successfully and all pass.
  * Two semantic segmentation integration tests have been failing on `master` and have been disabled for now.
* Verify that a code coverage report has been successfully uploaded to [Codecov](https://codecov.io/gh/azavea/raster-vision/commit/cdf0ab2d573a447903e52bb9b1ce41e6a309c557/graphs).
* Confirm that Release workflow logic covers all versions previously pushed by `.travis/deploy`.

Closes #1129
